src/datasets/jetstream.md

```diff
@@ -0,0 +1,243 @@
+Statistical summaries of telemetry data from experiments run in Mozilla
+products are provided by Jetstream. These summaries are published to
+BigQuery and serve both as the substrate for the result visualization
+platform and as a resource for data scientists.
+
+Jetstream runs as part of the nightly ETL job. The tables are published
+to the dataset `moz-fx-data-experiments.mozanalysis`.
+
+Experiments are analyzed using the concept of analysis windows. Analysis
+windows describe an interval marked from each client’s day of
+enrollment. The “day 0” analysis window aggregates data from the days
+that each client enrolled in the experiment. Because the intervals are
+demarcated from enrollment, they are not calendar dates; for some
+clients in an experiment, day 0 could be a Tuesday, and for others a
+Saturday.
+
+The week 0 analysis window aggregates data from each client’s days 0
+through 6, the week 1 window aggregates data from days 7 through 13, and
+so on.
+
+Clients are given a fixed amount of time, specified in Experimenter and
+often a week long, to enroll. Final day 0 results are available for
+reporting at the end of the enrollment period, after the last eligible
+client has enrolled, and week 0 results are available a week after the
+enrollment period closes. Results for each window are published as soon
+as complete data is available for all enrolled clients.
+
+The “overall” window, published after the experiment has ended, is a
+window beginning on each client’s day 0 that spans the longest period
+for which all clients have complete data.
+
+Jetstream computes statistics over several metrics by default, including
+for any features associated with the experiment in Experimenter. Data
+scientists can provide configuration to add additional metrics. Advice
+on configuring Jetstream can be found at Jetstream’s Github wiki.
+
+# Statistics tables
+
+The statistics tables contain statistical summaries of their
+corresponding aggregate tables. These tables are suitable for plotting
+directly without additional transformations.
+
+Statistics tables are named like:
+
+`statistics_<slug>_{day, week, overall}_<index>`
+
+A view is also created that concatenates all statistics tables for an
+experiment of a given period type, named like:
+
+`statistics_<slug>_{daily, weekly, overall}`
+
+Statistics tables have the schema:
+
+<table>
```

Comment:
Can we use a markdown table here (and elsewhere in the text), instead of HTML? I think that'd be much easier to review and maintain.

Reply:
This is what pandoc gave me when I asked for commonmark, which, sure enough, [does not spec tables](https://spec.commonmark.org/0.29/). ;_; I guess [pipe_tables](https://pandoc.org/MANUAL.html#tables) is the extension I need?

I should add this to the contributing guide.